templates/layout.html

```diff
@@ -54,6 +62,10 @@
 <script src="/vendor/bootstrap4-toggle/js/bootstrap4-toggle.js"></script>
 <script src="/vendor/apexcharts/dist/apexcharts.min.js"></script>
 <script src="/vendor/clipboard/dist/clipboard.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/sortablejs@latest/Sortable.min.js"></script>
```

Comment:
We want to move these files to the vendor directory.

Reply:
Hey just a thought on this - atm I am downloading vendor stuff via npm, and then copy/pasting them into vendor for ease of testing.

In the future, we should do some sort of compiling / minification that cuts out all the unused chunks of vendor stuff we're using, right? Can Flask do this or is there (yet) another library that will do this for us?